spec/factories.rb

```diff
@@ -138,6 +138,8 @@
 
   factory :reference do
     email_address { "#{SecureRandom.hex(5)}@example.com" }
+    name { "#{Faker::Name.first_name} #{Faker::Name.last_name}" }
+    relationship { 'My college tutor for 3 years' }
```

Comment:
Can you make this a random value so that tests can't rely on it? (Tests need to specify data explicitly, not rely on factories).